docs/INSTALL.md

```diff
@@ -7,20 +7,45 @@ This document contains information crucial for using and developing the project.
 An `.env` file with the following content is required at the root of the project (where `docker-compose.yml` is located): 
 
 ```
-# frontend
-FRONTEND_HOST = 'localhost'
-FRONTEND_PORT = '4000'
+# frontend - for docker only
+FRONTEND_HOST=0.0.0.0
+FRONTEND_PORT=4000
 
 # backend
-BACKEND_HOST = 'localhost'
-BACKEND_PORT = '8080'
-JWT_SECRET = 'secret'
-JWT_EXPIRES_IN = '7d'
+BACKEND_HOST=localhost
+BACKEND_PORT=8080
+JWT_SECRET=secret
+JWT_EXPIRES_IN=7d
+GOOGLE_EMAIL=mail@mail.com
+GOOGLE_PASSWORD=password
+GOOGLE_CLIENT_ID=id
+GOOGLE_CLIENT_SECRET=secret
+GOOGLE_CALLBACK_URL=http://localhost:4000/auth/google/callback
+SESSION_SECRET=secret
+COOKIE_SESSION_MAX_AGE=60000
+THROTTLE_TTL=60
+THROTTLE_LIMIT=5
 
 # database
-DB_NAME = 'mongo_obelisk'
-DB_USER = 'admin'
-DB_PASSWORD = 'password'
-DB_HOST = 'localhost'
-DB_PORT = '27017'
+DB_NAME=mongo_obelisk
+DB_USER=admin
+DB_PASSWORD=password
+DB_HOST=localhost
+DB_PORT=27017 # must be set to 27017
+```
+## Running with docker compose
+
+First, install `docker` and `docker compose`.
+
+Run the following commands at the root of the project (where `docker-compose.yml` is located):
+
+```
+docker compose build
```

Comment:
No need to run docker compose build when running for the first time - images will be build automatically. However when we are not sure whether we previously created the images we can simply run replacing two commands with one:
```
docker-compose up --build -d
``` 
--build = always rebuiling, -d = work i detach mode (run in the background)

Reply:
Yeah, I know all that.  I just thought it would be better to include more general version in docs.

Besides, `docker-compose` (aka Docker Compose v1) is deprecated ([link](https://github.com/actions/runner-images/issues/9557)) so it should be `docker compose up --build -d` following Docker Compose v2 syntax.